python/cuml/fil/fil.pyx

```diff
@@ -406,7 +414,20 @@ class ForestInference(Base):
         GPU array of length n_samples with inference results
         (or 'preds' filled with inference results if preds was specified)
         """
-        return self._impl.predict(X, preds)
+        return self._impl.predict(X, False, preds)
+
+    def predict_proba(self, X, preds=None):
+        """
+        X : array-like (device or host) shape = (n_samples, n_features)
```

Comment:
Please can you update the format of the documentation for predict_proba to show X and preds as the parameters and also describe what it returns. You can refer to the documentation for the predict function